scripts/amd/gemm/tune_gemm.py

```diff
@@ -115,6 +128,9 @@ def prune_configs(M, N, K, configs, elemBytes_a, elemBytes_b):
                 continue
             if num_warps < 4:
                 continue
+            # check if tiling is integer multiple of GEMM size because we have no boundary check
+            # if M % BLOCK_SIZE_M != 0 or N % BLOCK_SIZE_N != 0 or K % BLOCK_SIZE_K != 0:
+            #     continue
```

Comment:
I think we still don't have boundary check, or am I missing something?

Reply:
OK, we can turn on the check here. 
Just one concern that will make some not that "nice" shape have very few tuning configures and get bad performance.

Reply:
If you want to allow more tile sizes, you need to add mask to loads